src/applications/financial-status-report/components/SpouseGrossMonthlyIncomeInput.jsx

```diff
@@ -60,6 +61,7 @@ const SpouseGrossMonthlyIncomeInput = props => {
 
   const updateFormData = e => {
     e.preventDefault();
+    setGrossMonthlyIncome({ ...grossMonthlyIncome, dirty: true });
```

Comment:
Was there an issue with the data before adding this?

Reply:
Yeah, without this, you could skip the field without filling it in. This is part of the logic I copied over from the veteran version that was working.